modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/action/executor/test/ObjectActionExecutorRegistryImplTest.java

```diff
@@ -0,0 +1,249 @@
+ * details.
+ */
+
+package com.liferay.object.internal.action.executor.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.object.action.executor.ObjectActionExecutor;
+import com.liferay.object.action.executor.ObjectActionExecutorRegistry;
+import com.liferay.object.constants.ObjectActionExecutorConstants;
+import com.liferay.object.service.test.util.ObjectActionTestUtil;
+import com.liferay.portal.kernel.model.Company;
+import com.liferay.portal.kernel.security.auth.CompanyThreadLocal;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.CompanyTestUtil;
+import com.liferay.portal.kernel.util.HashMapDictionary;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceRegistration;
+
+/**
+ * @author Feliphe Marinho, Murilo Stodolni
+ */
+@RunWith(Arquillian.class)
+public class ObjectActionExecutorRegistryImplTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() throws Exception {
+		Bundle bundle = FrameworkUtil.getBundle(
+			ObjectActionExecutorRegistryImplTest.class);
+
+		_bundleContext = bundle.getBundleContext();
+
+		_companyId1 = CompanyThreadLocal.getCompanyId();
+
+		Company company = CompanyTestUtil.addCompany();
+
+		_companyId2 = company.getCompanyId();
+
+		_ootbObjectActionExecutors = Arrays.asList(
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_ADD_OBJECT_ENTRY),
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_GROOVY),
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_NOTIFICATION),
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_UPDATE_OBJECT_ENTRY),
+			_objectActionExecutorRegistry.getObjectActionExecutor(
+				ObjectActionExecutorConstants.KEY_WEBHOOK));
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		_unregisterObjectActionExecutors();
+	}
+
+	@Test
+	public void testGetObjectActionExecutors() {
+
+		// Available for all companies' object definitions
+
+		_assertObjectActionExecutors(
+			_objectActionExecutorRegistry.getObjectActionExecutors(
+				_companyId1, "Account"),
```

Comment:
@gabrielwas I noticed that we don't need to create object definitions and even companies for these tests because we don't need to check if them exist, so https://github.com/liferay-objects/liferay-portal/pull/2037/commits/6127927e0658759728805d194cc204362ae77077 and https://github.com/liferay-objects/liferay-portal/pull/2037/commits/50aa7d5f062acebe6d7631981f5ded5b8dfe49f5